### ISSUE TITLE
Node restart required after deleting a node

### ISSUE THREAD

**Issue by jimboca** [UniversalDevicesInc/polyglot-v2#13]
I delete a node in my nodeserver from the Polyglot UI and then rerun discover for my nodeserver but the deleted node doesn't come back.  I have to restart Polyglot to get the deleted node to add.

**Comment by exking** [comment]
You mean restart the node server? Nodeserver must still have node address in self.nodes so addNode() is not called?

**Comment by jimboca** [comment]
Yes, that's what I said :-)  and yes, it must still be in self.nodes, it may work if I set update=True, but the nodeserver has no way of knowing that.

**Comment by exking** [comment]
Sounds like an enhancement, Polyglot must send a "node deleted" MQTT event down to polyinterface and polyinterface must remove address from self.nodes.

**Comment by Einstein42** [comment]
Polyglot does send a config() down the pipe to nodeserver on deletion which in turn updates the self._nodes dictionary. Is this a polyglot issue or a polyinterface issue?
